datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -357,6 +357,8 @@ def start_agent(self):
         command = [
             'docker',
             'run',
+            # Remove it when stopped
+            '--rm',
```

Comment:
Do I understand this correctly, that it will remove the container in these cases:

1. after we run `ddev env stop` (-> `docker stop` -> `--rm` flag takes effect)
2. if the container or the docker daemon or the computer crash (assuming docker has the chance to "clean up")

Reply:
Yes

Reply:
Can you think of a scenario where this would be undesirable behavior? I can't but I just wanna make sure we don't break someone's workflow...

Reply:
I don't think so. This is the agent container, not the service itself so apart from the config file, there's nothing to save AFAIK